birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -96,12 +96,18 @@ if [ -z "$WEAVER_CURL_IMAGE" ]; then
     WEAVER_CURL_IMAGE="curlimages/curl:7.87.0"
 fi
 
+# POSIX portable RNG if RANDOM does not exist on the current shell
+RANDOM_NUMBER=${RANDOM:-$(tr -dc 0-9 < /dev/urandom | head -c 5)}
+
 # To know when a docker run was started in case it hangs.
-DOCKER_RUN_TAG="`date -Isecond | sed 's/:/_/g' | sed 's/+/p/g'`_$RANDOM"
+DOCKER_RUN_TAG="weaver_post_curl_`date -Isecond | sed 's/:/_/g' | sed 's/+/p/g'`_${RANDOM_NUMBER}"
```

Comment:
I intentionally not put `weaver_post_curl_` in `DOCKER_RUN_TAG` so the same tag can be used for another docker run invocation in the future, if this script needs more than `curl` later.

But can leave this as-is for now.  Later if more than `curl` via `docker run` is needed, you can make this `DOCKER_RUN_TAG` more generic.

Reply:
I thought that might be the case, but since the random value was fixed in `DOCKER_RUN_TAG`, the full date/rng command would have to be duplicated as well anyway.

Reply:
I intended one `DOCKER_RUN_TAG` per invocation of each script.  The `RANDOM` part was in case 2 scripts start exactly at the same second so the timestamp is not different enough.  But this is small details.  Either way is fine with me.